actor-profile.js

```diff
@@ -0,0 +1,145 @@
+import { db } from './dbInstance.js'
+
+class ActorProfile extends HTMLElement {
+  static get observedAttributes () {
+    return ['url']
+  }
+
+  constructor () {
+    super()
+    this.url = ''
+  }
+
+  connectedCallback () {
+    this.url = this.getAttribute('url')
+    this.fetchAndRenderActorProfile(this.url)
+  }
+
+  async fetchAndRenderActorProfile (url) {
+    const actorInfo = await db.getActor(url)
+    console.log(actorInfo)
+    if (actorInfo) {
+      this.renderActorProfile(actorInfo)
+    }
```

Comment:
we should have an error state. Maybe we could have a standard component for this?